src/main/java/org/broadinstitute/gatk/nativebindings/pdhmm/ReadDataHolder.java

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.gatk.nativebindings.pdhmm;
+
+/**
+ * Struct used to copy references to byteArrays from reads to the PDHMM compute kernel.
+ */
+public final class ReadDataHolder {
+    public byte[] readBases = null;
+    public byte[] readQuals = null;
+    public byte[] insertionGOP = null;
```

Comment:
Are these numbers obviously meaningful to you James or would this need comments?

Reply:
this looks good to me